packages/components/time_region_value/src/node_modules/stores/layout.js

```diff
@@ -1,13 +1,25 @@
+import * as _ from 'lamb';
 import {derived, writable} from 'svelte/store';
 import {linearScale} from 'yootils';
+import {screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
 
 import {yearExtentStore} from './data';
 
-const safetyDefault = {left: 120, right: 80};
-const safetyStore = writable(safetyDefault);
-export const resetSafetyStore = () => {
-	safetyStore.set(safetyDefault);
-}
+/* responsive */
+
+export const isSmallScreen = derived(screen, $screen =>
+	$screen && ($screen.sizeFlags.small && !$screen.sizeFlags.medium)
+);
+export const screenClasses = derived(screen, s => s?.classes);
+
+/* safety */
+
+const smallSafety = {left: 40, right: 40};
+const largeSafety = {left: 120, right: 80};
+export const safetyStore =
+	derived(isSmallScreen, isSmall => isSmall ? smallSafety : largeSafety);
```

Comment:
I think it might be a bit easier to read this way:
```
export const safetyStore = derived(isSmallScreen, isSmall => isSmall
	? smallSafety
	: largeSafety
);
```